src/sentry/static/sentry/app/components/events/contexts/gpu.jsx

```diff
@@ -0,0 +1,65 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import ContextBlock from 'app/components/events/contexts/contextBlock';
+import {formatBytes} from 'app/utils';
+
+const megaByteInBytes = 1048576;
+
+class GpuContextType extends React.Component {
+  static propTypes = {
+    alias: PropTypes.string.isRequired,
+    data: PropTypes.object.isRequired,
+  };
+
+  formatMemory = memory_size => {
+    if (!Number.isInteger(memory_size) || memory_size <= 0) {
+      return null;
+    }
+
+    // 'usable_memory' is in defined in MB
+    return formatBytes(memory_size * megaByteInBytes);
+  };
+
+  render() {
+    let {
+      id,
+      name,
+      version,
+      vendor_name,
+      vendor_id,
+      memory_size,
+      npot_support,
+      multi_threaded_rendering,
+      api_type,
+      ...data
+    } = this.props.data;
+
+    let memory = this.formatMemory(memory_size);
+    let knownData = [
+      ['?Name', name],
+      ['?Version', version],
+      ['?Vendor', vendor_name],
+      ['?Memory', memory],
+      ['?NPOT Support', npot_support],
+      ['?Multi-Thread rendering', multi_threaded_rendering],
+      ['?API Type', api_type],
+    ];
+
+    if (id > 0) {
+      knownData.unshift(['?Vendor Id', vendor_id]);
+    }
+
+    if (id > 0) {
```

Comment:
This condition could be combined with the one above it.

Reply:
You just caught a bug there! I should be checking for: `vendor_id `.
This is due to Unity serialization not handling `int?`, so type is not nullable and `0` will be sent :\
It will be the case until we get away from `Unity.JsonUtility`